### PR TITLE
Blog Tags, Auth Post, Safe Decoding in Go Post

### PR DESCRIPTION
**Pull request by blainsmith**
### Tags

Everything is working as expected. The only thing to remember is to make sure you capitalize the tag when putting it into the frontmatter of the blog posts.

**Do This**
```toml
[taxonomies]
tags = ["Company"]
```

**Don't Do This**
```toml
[taxonomies]
tags = ["company"]
```

### Auth Post

This isn't completed yet and I am happy to choose a different topic, but I thought this could bridge into Engineering and Product as our first post there to support the tags. Additionally, this topic is good for our target customer should know about and we can get into the details a bit to flex our knowledge muscles on it since, as I said before, every app needs auth regardless of what we build.

I am keeping this in "Draft" for now until the blog post is done, but wanted to get some initial eyes on it to get a feel for the auth topic.

### Safer Decoding in Go

screencapture-127-0-0-1-1111-blog-safer-binary-decoding-in-go-2024-06-03-18_48_20

### CODE REVIEW

**Review by johnluther — COMMENTED** [review]
Added the draft tag to the new blog post. Cool topic, looking forward to seeing the full draft!

File: content/blog/auth-accounts-and-identity-management-explained.md
Comment (erkattak) [inline content/blog/auth-accounts-and-identity-management-explained.md:1]: This could be a whole series. Email verification, forgotten passwords, etc.

Comment (blainsmith) [reply content/blog/auth-accounts-and-identity-management-explained.md:1]: Yup which we should definately do as a longer and planned series. That is why I tabled this one and went with the other to get something out sooner.

**Comment by erkattak** [comment]
I would just say probably move the example code for the post into a separate repo, or gist. I can see this repo getting fairly big with example code if we do it this way

**Comment by johnluther** [comment]
Looks good to me!

**Pull request merged**